drivers/lcd/st7567.c

```diff
@@ -729,7 +729,9 @@ static int st7567_getpower(struct lcd_dev_s *dev)
 {
   struct st7567_dev_s *priv = (struct st7567_dev_s *)dev;
   DEBUGASSERT(priv);
-  ginfo("powered: %s\n", st7567_powerstring(priv->powered));
+
+  /* ginfo("powered: %s\n", st7567_powerstring(priv->powered)); */
```

Comment:
should we remove it directly?

Reply:
i have no strong opinion either ways.

Reply:
Since the code can't compile, it's better to remove it.

Reply:
done